tests/tasks/assert_certificate_parameters.yml

```diff
@@ -0,0 +1,35 @@
+---
+- name: Verify certificate exists in the correct location
+  file:
+    path: "{{ cert['path'] }}"
+    state: file
+  register: result
```

Comment:
do you need the register here?

Reply:
did you mean to use `stat` instead of `file`?  Something that would throw an error if the file does not exist?

Reply:
Actually I wasn't sure at all about what to use here. In other PR I've used file and registered the result to verify with assert. It seems like only `stat` would be able to do the job by itself, right?
I'll updated it.

Reply:
`stat` only added the "assertion" capabilities starting from ansible 2.9.

Reply:
Using `fail` module is preferable then using `assert` module?
`is exists` is preferable over `stat.exists`. Note that we still need stat to check owner/group in next PRs.

Reply:
> Using `fail` module is preferable then using `assert` module?

No.  Either is fine.

> `is exists` is preferable over `stat.exists`. Note that we still need stat to check owner/group in next PRs.

Ah, if you need the result of `stat` then keep it.

Reply:
I also have register in the stat task in the logging role CI-tests. Should I also consider getting rid of the register?
Regarding the ansible version, I see 2.9 is available on RHEL-7 and 8 as well as Fedora's. Do we still have to worry about older versions of ansible?

Reply:
@seocam do you have a link that says "assert" is only supported in `stat` in ansible 2.9 and later?

Reply:
As a general rule in Red Hat we currently should support 2.7+ (in a few weeks only 2.8+)[1]. I'm not sure if that applies to linux-system-roles as well.

[1] https://access.redhat.com/support/policy/updates/ansible-engine

Reply:
> As a general rule in Red Hat we currently should support 2.7+ (in a few weeks only 2.8+)[1]. I'm not sure if that applies to linux-system-roles as well.

It applies - we support ansible 2.7, 2.8 and 2.9
.
> 
> [1] https://access.redhat.com/support/policy/updates/ansible-engine

Reply:
@seocam what are those "assertion" capabilities starting from ansible 2.9? I do not see any assertion mentioned in https://docs.ansible.com/ansible/latest/modules/stat_module.html

Reply:
> in a few weeks only 2.8+

That's weird, I have supposed that 2.7 will be supported until the release of 2.10.

Reply:
> `is exists` is preferable over `stat.exists`

no, `is exists` checks for paths on the controller and is totally unrelated to the return value of `stat`. https://docs.ansible.com/ansible/latest/user_guide/playbooks_tests.html#testing-paths

Reply:
> > in a few weeks only 2.8+
> 
> That's weird, I have supposed that 2.7 will be supported until the release of 2.10.

I asked @maxamillion for clarification.